crates/test-utils/src/chain/test_chain.rs

```diff
@@ -1016,19 +1023,35 @@ impl TestChain {
             logs[0].clone().hyperdrive
         };
 
-        // Transfer ownership of the base token, factory, vault, and lido to the
-        // admin address now that we're done minting tokens and updating the
-        // configuration.
+        // Add the 4626 Hyperdrive instance and the Lido Hyperdrive instance
+        // to the registry contract.
+        hyperdrive_registry
+            .set_hyperdrive_info(vault.address(), uint256!(1))
+            .send()
+            .await?;
+        hyperdrive_registry
+            .set_hyperdrive_info(lido.address(), uint256!(1))
+            .send()
+            .await?;
+
+        // Transfer ownership of the base token, factory, vault, lido,
+        // and the registry to the admin address now that we're done
+        // minting tokens and updating the configuration.
         base.transfer_ownership(config.admin).send().await?;
         vault.transfer_ownership(config.admin).send().await?;
         lido.transfer_ownership(config.admin).send().await?;
         factory.update_governance(config.admin).send().await?;
+        hyperdrive_registry
+            .update_governance(config.admin)
+            .send()
+            .await?;
```

Comment:
One problem that we may run into is that the devnet `admin` defaults to `address(0)`, which would prevent the registry from being updated on `devnet`. I think we'll want to update `.github/workflows/devnet.yaml` so that it has an `ADMIN` environment variable similar to `.github/workflows/testnet.yaml`. We can use a Github variable or just put a comment with the private key in the workflow since this is used for developer purposes.

Reply:
With this in mind (and looking at the review checklist), let's add some tests that verify that we can actually update the registry.